ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/CommitTxnHandler.java

```diff
@@ -110,9 +109,9 @@ private void createDumpFileForTable(Context withinContext, org.apache.hadoop.hiv
 
   private List<WriteEventInfo> getAllWriteEventInfo(Context withinContext) throws Exception {
     String contextDbName = StringUtils.normalizeIdentifier(withinContext.replScope.getDbName());
-    RawStore rawStore = HMSHandler.getMSForConf(withinContext.hiveConf);
     List<WriteEventInfo> writeEventInfoList
-            = rawStore.getAllWriteEventInfo(eventMessage.getTxnId(), contextDbName, null);
+            = withinContext.db.getMSC().getAllWriteEventInfo(
+                new GetAllWriteEventInfoRequest(eventMessage.getTxnId(), contextDbName, null)).getWriteEventInfos();
```

Comment:
GetAllWriteEventInfoRequest should have a constructor without table name if its optional param